packages/google-pay-integration/src/gateways/google-pay-gateway.ts

```diff
@@ -389,7 +389,7 @@ export default class GooglePayGateway {
             company,
             address1,
             address2: `${address2} ${address3}`.trim(),
-            city,
+            city: city || stateOrProvinceCode,
```

Comment:
I am not sure that we can fix similar behaviour in this way. Because `city` is not `stateOrProvinceCode`. Were you aligned it with someone?

Reply:
`city` is not the same as `stateOrProvinceCode`. But if `city` field is empty we need to fill it with some data. In our case `stateOrProvinceCode` fits the best, because in the GP documentation it means `A country subdivision, such as a state or province.`.